starters/apps/playground/src/routes/layout.tsx

```diff
@@ -7,17 +7,6 @@ import Footer from "../components/starter/footer/footer";
 
 import styles from "./styles.css?inline";
 
-export const onGet: RequestHandler = async ({ cacheControl }) => {
-  // Control caching for this request for best performance and to reduce hosting costs:
-  // https://qwik.dev/docs/caching/
-  cacheControl({
```

Comment:
maybe we can leave a comment with a link to the blog post or docs on the matter, something like: "It's a good idea to set your cache for X, here's how to do it..."

So it will be part of the starters